changelog.d/13766.bugfix

```diff
@@ -0,0 +1 @@
+Make sequence cache_invalidation_stream_seq begins at 2.
```

Comment:
Where possible, we like to mention the version of Synapse that introduced a bug, or call it long-standing if it's been around for a long time / since the code was written.

```suggestion
Fix a long-standing bug where the `cache_invalidation_stream_seq` sequence would begin at 1 instead of 2.
```